src/index.js

```diff
@@ -4,12 +4,12 @@ import {BrowserRouter} from 'react-router-dom';
 
 import './index.css';
 import App from './App';
-import AppV2 from './AppV2';
+// import AppV2 from './AppV2';
```

Comment:
(temporarily exclude AppV2 until JS is fully merged)

Reply:
Why do we need this?  If the v2/ routes are a little busted that's fine by me.

Reply:
AppV2 is @manuel-calavera rewrite, which needs updating & should hopefully land soon